suitcase/utils/tests/conftest.py

```diff
@@ -211,3 +211,30 @@ def _example_data_func(ignore=[]):
         return generate_data(plan_type(ignore), ignore=ignore)
 
     return _example_data_func
+
+
+@pytest.fixture(params=['test-', 'scan_{uid}-'],
+                scope='function')
+def file_prefix_list(request):  # noqa
+    '''Returns a list file_prefixes to be formatted for testing.
```

Comment:
The description of this function does not match the return type.

Reply:
fixed